src/parser.rs

```diff
@@ -1017,10 +1017,31 @@ impl Parser {
             ColumnOption::Unique { is_primary: false }
         } else if self.parse_keyword("REFERENCES") {
             let foreign_table = self.parse_object_name()?;
-            let referred_columns = self.parse_parenthesized_column_list(Mandatory)?;
+            let referred_columns = self.parse_parenthesized_column_list(Optional)?;
```

Comment:
Please add a comment, e.g. `// PostgreSQL allows omitting the column list to refer to the foreign table by its PK`